spec/controllers/errors_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+describe ErrorsController do
+  describe "GET #show" do
+
+    before :all do
+      secrets = Rails.application.secrets
+      @secret_key_base = secrets.secret_key_base
+      @secret_machine_fqdn = secrets.machine_fqdn
+      @secret_encryption_private_key_path = secrets.encryption_private_key_path
+      @secret_portus_password = secrets.portus_password
+    end
+
+    before :each do
+      secrets = Rails.application.secrets
+      secrets.secret_key_base = @secret_key_base
+      secrets.machine_fqdn = @secret_machine_fqdn
+      secrets.encryption_private_key_path = @secret_encryption_private_key_path
+      secrets.portus_password = @secret_portus_password
+    end
+
+    after :all do
+      secrets = Rails.application.secrets
+      secrets.secret_key_base = @secret_key_base
+      secrets.machine_fqdn = @secret_machine_fqdn
+      secrets.encryption_private_key_path = @secret_encryption_private_key_path
+      secrets.portus_password = @secret_portus_password
+    end
```

Comment:
Do we really need that? Is that because otherwise the other tests are screwed up?